docs/source/blocks/index.md

```diff
@@ -1,10 +1,10 @@
 ---
 myst:
   html_meta:
-    "description": "Volto features the Pastanaga Editor Engine, allowing you to visually compose a page using blocks."
-    "property=og:description": "Volto features the Pastanaga Editor Engine, allowing you to visually compose a page using blocks."
-    "property=og:title": "Blocks"
-    "keywords": "Volto, Plone, frontend, React, blocks"
+    'description': 'Volto features the Pastanaga Editor Engine, allowing you to visually compose a page using blocks.'
+    'property=og:description': 'Volto features the Pastanaga Editor Engine, allowing you to visually compose a page using blocks.'
+    'property=og:title': 'Blocks'
+    'keywords': 'Volto, Plone, frontend, React, blocks'
```

Comment:
Please revert all changes of `"` to `'`. And configure your editor to be less aggressive in making changes.

This PR cannot be accepted with wholesale changes to whitespace and quotation marks.